RecoLocalCalo/Configuration/python/RecoLocalCalo_Cosmics_cff.py

```diff
@@ -74,20 +74,27 @@
 )
 
 
-run2_HE_2017.toReplaceWith(hbheprereco, _phase1_hbheprereco )
+run2_HCAL_2017.toReplaceWith(hbheprereco, _phase1_hbheprereco )
 run2_HF_2017.toReplaceWith(hfreco, _phase1_hfreco )
 
 hfprereco = _hcalLocalReco_cff.hfprereco.clone(
     sumAllTimeSlices = cms.bool(True)
 )
 
+hbheplan1 = _hcalLocalReco_cff.hbheplan1.clone()
```

Comment:
shouldn't the _hcalLocalReco_cff's hbheplan1 have a different name to have meaningful copy?
if some place else loads hcalLocalReco_cff, it will be overwritten again